velox/connectors/hive/SplitReader.cpp

```diff
@@ -294,7 +294,8 @@ void SplitReader::createReader() {
     fileHandleCachePtr = fileHandleFactory_->generate(
         hiveSplit_->filePath,
         hiveSplit_->properties.has_value() ? &*hiveSplit_->properties : nullptr,
-        fsStats_ ? fsStats_.get() : nullptr);
+        fsStats_ ? fsStats_.get() : nullptr,
```

Comment:
There is a filehandle cache and this caches by path. Which means that if 2 users query the same path only the first users session properties will be shown